pkg/resourcemonitor/podresourcesscanner.go

```diff
@@ -198,7 +203,27 @@ func (resMon *PodResourcesScanner) Scan() ([]PodResources, error) {
 
 	}
 
-	return podResData, nil
+	retVal := ScanResponse{
+		PodResources: podResData,
+		Attributes:   v1alpha2.AttributeList{},
+	}
+
+	if resMon.podFingerprint && len(retVal.PodResources) > 0 {
+		var status podfingerprint.Status
+		podFingerprintSign, err := computePodFingerprint(retVal.PodResources, &status)
+		if err != nil {
+			klog.Errorf("podFingerprint: Unable to compute fingerprint %v", err)
+		} else {
+			klog.Infof("podFingerprint: " + status.Repr())
```

Comment:
nit: no string formatting is used here so either

```suggestion
			klog.Info("podFingerprint: " + status.Repr())
```

or 
```suggestion
			klog.Infof("podFingerprint: %s", status.Repr())
```

Reply:
Thanks for the comment and the suggestions.